cider/exec/template/ScalarExprVisitor.h

```diff
@@ -147,6 +143,10 @@ class ScalarExprVisitor {
     if (agg) {
       return visitAggExpr(agg);
     }
+    const auto string_oper = dynamic_cast<const Analyzer::StringOper*>(expr);
```

Comment:
will this be called? can you point out code path?

Reply:
mainly in substraitToEu's filter part

ctx_element->accept(rel_visitor_ptr);
=>
FilterQualContext::accept
=>
FilterRelVisitor::visit
=>
and after substrait2Analyzer (toAnalyzerExpr)
=>
fold_expr(cider_expr.get())
=>
visitor.visit()